composer.json

```diff
@@ -18,7 +18,7 @@
 	],
 
 	"require": {
-		"php": ">=5.6.0",
+		"php": ">=7.0",

```

Comment:
But the class pass successfully from the PHP Compatibility Coding Standards for PHP CodeSniffer!
`phpcs -p Arabic.php --standard=PHPCompatibility --runtime-set testVersion 5.6-`

Then I noticed that you may referring to the [first line](https://github.com/khaled-alshamaa/ar-php/blob/245b7f03103617addddd19d9a7bce31334d9ea13/tests/ArabicTest.php#L1) in the ArabicTest.php script for the PHPUnit Testing Framework. 

Well, that line can be dropped safely with no effect on any of the library functionalities at all, so we keep poor developers who still maintain PHP 5.6 projects supported in our eco-system ;-)